pkg/apis/pipeline/v1/pipeline_types.go

```diff
@@ -228,6 +228,16 @@ type PipelineTask struct {
 	// Refer Go's ParseDuration documentation for expected format: https://golang.org/pkg/time/#ParseDuration
 	// +optional
 	Timeout *metav1.Duration `json:"timeout,omitempty"`
+
+	// PipelineRef is a reference to a pipeline definition
+	// Note: PipelineRef is in preview mode and not yet supported
+	// +optional
+	PipelineRef *PipelineRef `json:"pipelineRef,omitempty"`
```

Comment:
@lbernick do we introduce these API changes in `v1beta1` as well? Or this is just limited to `V1`?

Reply:
I think we should only bring new features into v1.

Reply:
That's a good question, but as long as we "support" both, we may want to add them to both ?
But I can also get behind just introducing them in v1 🙃

Reply:
Thanks @vdemeester and @lbernick 

Yeah I am trying to come up with the best way to handle this. Yup, we can introduce them to both. The advantage of introducing to both:

- Collect feedback from the pipeline authors who are currently creating specifications using `v1beta1` to help us discovering any unknown issues.
- Pipelines in pipelines was [designed](https://github.com/tektoncd/community/commit/fc1dd1065a7325cee6b3df1b36f6bedb316d7d8d) more than a year back in June 2022 -  while it was [requested](https://github.com/tektoncd/pipeline/issues/2134) in March, 2020. As a community, we haven't been able to prioritize and implement it until now, more than three years since it was requested.

Reply:
I think we introduce this feature to both V1 and v1beta1 because the FR was raised when we only have v1beta1 is a good justification?

Following that, we could say we support new features in v1 only if the FR is raised after we switched to v1.

Reply:
Thank you @lbernick @vdemeester @QuanZhang-William, I have added these changes to `v1beta1` as well.